e2e/BUILD.bazel

```diff
@@ -51,15 +51,28 @@ e2e_integration_test(
     tags = ["no-bazelci-windows"],
 )
 
-e2e_integration_test(
-    name = "e2e_jasmine",
+[e2e_integration_test(
+    name = "e2e_jasmine_%s" % jasmine_version.replace(".", "_"),
     npm_packages = {
         "//packages/jasmine:npm_package": "@bazel/jasmine",
     },
+    # use these package.json packages instead
+    package_json_substitutions = {
+        "jasmine": jasmine_version,
+        "jasmine-core": jasmine_version,
+    },
     # TODO: figure out why this fails on Windows since setting
     # symlink_node_modules to False in the test WORKSPACE
     tags = ["no-bazelci-windows"],
-)
+    workspace_root = "jasmine",
+) for jasmine_version in [
```

Comment:
nice, thanks for considering the back-compat case.
Too bad we didn't land this in time for 5.0, we could have just dropped support for older jasmine in a breaking change.

Do you think we should leave a `TODO(6.0)` here to remove old support for old Jasmine?

Reply:
Sure